omod/src/test/java/org/openmrs/module/fhir2/providers/ObservationFhirResourceProviderTest.java

```diff
@@ -3,7 +3,6 @@
  * v. 2.0. If a copy of the MPL was not distributed with this file, You can
  * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
  * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
- *
```

Comment:
Please remove this from this PR.

Reply:
Finished

Reply:
Anything missing @ibacher @corneliouzbett